src/pull-request-data-provider.js

```diff
@@ -22,19 +22,18 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-const { graphql } = require("@octokit/graphql");
-const { createAppAuth } = require("@octokit/auth-app");
+const {graphql} = require('@octokit/graphql');
```

Comment:
I strongly disagree with the fact that we use octokit library as there was no updates since 4 months and apollo is strongly maintained/used in the JS community, we can still use https://github.com/apollographql/apollo-link#readme with a fetch polyfill !

Wdyt @PierreRambaud @eternoendless 

I know that's not related to this PR, but it's pretty weird to be honest

Reply:
Probot is already using octokit/graphql
https://github.com/probot/probot/blob/master/package.json#L55

But I can change our used lib to apollographql

Reply:
It's not related to the PR, wait for @eternoendless and @PierreRambaud advice

Reply:
@NeOMakinG `@octokit/graphql` has been updated one week ago on GitHub whereas apollo was one month ago. Not having a release doesn't mean the project is not maintained. And because probot is also using this library, we should reuse the same library :thinking:

Reply:
Well I should have express that differently, Apollo is a reference used by most of JS community members and maintained by 10x more customers than octokit, but ok.